improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -222,11 +220,11 @@ def _add_bounds_to_percentiles_and_forecast_at_percentiles(
                     "as new bounds."
                 )
                 warnings.warn(warn_msg)
-                if upper_bound < forecast_at_percentiles_with_endpoints.max():
-                    upper_bound = forecast_at_percentiles_with_endpoints.max()
-                if lower_bound > forecast_at_percentiles_with_endpoints.min():
-                    lower_bound = forecast_at_percentiles_with_endpoints.min()
-                forecast_at_percentiles_with_endpoints = concatenate_2d_array_with_2d_array_endpoints(
+                if upper_bound < forecast.max():
+                    upper_bound = forecast.max()
+                if lower_bound > forecast.min():
+                    lower_bound = forecast.min()
```

Comment:
Raised #1506 to cover this if we want

Reply:
If someone could grant me absolution for the patch failing here - @dmentipl or @tjtg? - I'll override the checks for merging.

Reply:
Now that I've approved it, you should be fine to merge. I absolve you.